packages/bundle-source/src/index.js

```diff
@@ -51,7 +55,12 @@ export default async function bundleSource(
     if (isEntry) {
       entrypoint = fileName;
     }
-    sourceBundle[fileName] = code;
+    // Rewrite apparent import expressions so that they don't fail under SES.
+    // We also do apparent HTML comments.
+    const defangedCode = code
+      .replace(IMPORT_RE, '$1notreally')
+      .replace(HTML_COMMENT_RE, '<->');
```

Comment:
Since these may be within quoted strings or template literals, we should at least emit a warning with line numbers. My preference would be an error by default, with some option needed to proceed despite detecting one of these.

Reply:
I don't want to proceed despite detecting it, or else that would admit code that actually is dangerous!

I want to disable such dangerous code in such a way that old-SES never knows what happened.  In the future, we will translate import expressions.  As you taught me, HTML comments are a problem no matter what (regardless of whether they appear within a JS comment or not) and need to be defanged.

This change is just needed for the expedience of allowing the import of types between source files (within JSDoc comments).  It accumulates technical debt, but I would rather go into debt to make a deadline, and pay it back when we get to new-SES.

Reply:
> Since these may be within quoted strings or template literals, we should at least emit a warning with line numbers.

I can do that.

Reply:
> I can do that.

Oh, but that would generate a bunch of warnings for Typescript imports that are totally harmless and would not be evaluated.